### PR TITLE
feat(copy): add --no-clobber to not overwrite files if they exist and have the same size

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: src/commands/copy/copy.ts [hunk 1]
```diff
@@ -27,6 +29,18 @@ export const commandCopy = command({
   args: {
     config,
     verbose,
+    force: flag({
```
[the remark this answers is not in the document]
Comment (blacha) [reply src/commands/copy/copy.ts:32]: Lets you choose what option you want do to, I think `--no-clobber` is what we want most of the time, and then if something goes wrong `--force` would let us fix things.

**Review by amfage — APPROVED** [review]

File: src/commands/copy/copy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (amfage) [reply src/commands/copy/copy.ts:32]: Could just default to a `no-clobber` mode and use `--force` if necessary?

[the remark this answers is not in the document]
Comment (paulfouquet) [reply src/commands/copy/copy.ts:32]: It is default to `no-clubber`

[the remark this answers is not in the document]
Comment (blacha) [reply src/commands/copy/copy.ts:32]: It shouldnt be :P fixed in https://github.com/linz/argo-tasks/pull/103/commits/b6b18a9e24812b273cb1622e20e6ffc951ee4892

[the remark this answers is not in the document]
Comment (paulfouquet) [reply src/commands/copy/copy.ts:32]: In that case are we not throwing an error every time the file already exists?

**Pull request merged**